derived_images/mariadb/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM opensuse:leap
```

Comment:
I'd rather write:

```
FROM opensuse/leap:15.0
```

@Vogtinator told me that this is the image to be used for Leap.

Reply:
It depends on what you want - currently "leap:latest" == "leap:15" == "leap:15.0".
It'll change when 16 and 15.1 get released, respectively.

Reply:
"opensuse:leap" currently references 42.3.

Reply:
Oh, gotcha. Then I'd say to specify the tag always, to avoid surprises.

Reply:
"opensuse:leap" gets only important security fixes for 42.3 but is otherwise dead.